exchanges/okx/okx_websocket.go

```diff
@@ -538,10 +538,10 @@ func (ok *Okx) handleSubscription(operation string, subscriptions subscription.L
 
 	channels = append(channels, authChannels...)
 	if operation == operationUnsubscribe {
-		return ok.Websocket.RemoveSubscriptions(channels...)
+		return ok.Websocket.RemoveSubscriptions(ok.Websocket.Conn, channels...)
```

Comment:
Looks like it's working for me.
```
go run ./... pair disable --exchange='okx' --pairs="ape-usdc" --asset=spot
{
 "status": "success"
}
```

```
go run ./... pair enable --exchange='okx' --pairs="ape-usdc" --asset=spot
{
 "status": "success"
}
```

Originally there was no differentiation between authenticated and unauthenticated connections. One of the reasons why I opted for a nil when I opened this PR because it was specifically not going to be matched. It should just be defaulting to what it was doing prior.